### DISCUSSION TITLE
Provide option of removing "Execute cell" button.

### DISCUSSION THREAD

**Discussion by baiguoname** [microsoft/vscode-jupyter#10143] [Questions and Answers]
The "Execute cell" button in front of each cell, is so anonying to me. Please provide a option to remove this buttton.

**Comment by rchiodo** [comment]
This button isn't actually provided by the jupyter extension. I recommend adding an issue to vscode itself: https://github.com/microsoft/vscode/issues

**Reply by baiguoname** [reply]
![image](https://github.com/microsoft/vscode-jupyter/assets/52092777/dda134a0-133e-40bc-9e3b-62bef4b7bf28)
It seems to me that the code from https://github.com/microsoft/vscode-jupyter/blob/main/src/webviews/webview-side/react-common/image.tsx line 137 would be the place the button loaded.
